packages/shared/lpc/tests/empty_catalog.test.ts

```
@@ -0,0 +1,63 @@
+// packages/shared/lpc/tests/empty_catalog.test.ts
+//
+// AC-4: The empty catalog never crashes a consumer.
+
+import { describe, expect, test } from 'bun:test';
+import { buildLpcCatalog } from '../src/lib/build_catalog.ts';
+import { resolveLpcAppearance, projectLpcCatalog, DEFAULT_LPC_SLOT_FALLBACKS, LPC_SLOT_ORDER } from '../src/lib/appearance.ts';
```

Biome (2.5.7)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)